generate-snapshot.test.ts

```diff
@@ -22,9 +22,16 @@ describe("generateSnapshot", () => {
     });
 
     it("creates a snapshot with a single user", () => {
-        when(getUserValues).calledWith("user1").thenReturn("foo");
+        const user1: User = {
+            id: 1,
+            username: "abc123",
+            firstName: "fName1",
+            lastName: "lName1",
+            studentId: "123456789",
+        };
 
-        generateSnapshot(["user1"]);
+        when(getUserValues).calledWith(user1).thenReturn("foo");
+        generateSnapshot([user1]);
```

Comment:
Really minor thing - but I think it'd be good to have a line break between these two lines. The `when(getUserValues)...` line is part of the "arrange" section of the test, and the `generateSnapshot()...` line is part of the "act" section of the test, and I think it's nice to keep the sections separated if possible. When we learn about Grails, we'll see that the test framework there has specific keywords to keep the arrange/act/assert sections separate, which is pretty useful, but here in Vitest there's nothing like that, just blank lines.